config/initializers/faraday.rb

```diff
@@ -0,0 +1,17 @@
+ActiveSupport::Notifications.subscribe('request.faraday') do |name, starts, ends, _, env|
+  url = env[:url]
+  http_method = env[:method].to_s.upcase
+  duration_seconds = ends - starts
+  metadata = {
+    http_method: http_method,
+    host: url.host,
+    path: url.path,
+    duration_seconds: duration_seconds,
+    status: env.status,
+    service: "#{http_method}-#{url.host}#{url.path}",
```

Comment:
do we need to worry about paths containing IDs and things that make grouping harder? ex some of the document ID in the acuant API?

Reply:
Ah, yes, we do. I didn't realize there was a dynamic part of the path for some of these. My backup plan was to see if we could add some friendlier names and group that way, and that's necessary now so I'll make some edits to do that.

Reply:
Maybe we add our own Faraday middleware with like a "request_name" or "reporting_name" attribute? or mabe we just get in the habit of adding a special `X-Login-Name` header or something to each request?

Reply:
I like the middleware route. Do you know if its okay to log document IDs from Acuant? If not, I can remove path from the logs.

Reply:
Yeah logging the document IDs in the path should be fine, they're just references that aren't PII themselves